nni/retiarii/converter/graph_gen.py

```diff
@@ -590,7 +581,13 @@ def convert_module(self, script_module, module, module_name, ir_model):
         if original_type_name in MODULE_EXCEPT_LIST:
             pass  # do nothing
         elif original_type_name == OpTypeName.LayerChoice:
-            m_attrs = self._handle_layerchoice(module)
+            graph = Graph(ir_model, -100, module_name, _internal=True)  # graph_id is not used now
+            candidate_name_list = [f'layerchoice_{module.label}_{cand_name}' for cand_name in module.names]
+            for cand_name, cand in zip(candidate_name_list, module):
+                cand_type = '__torch__.' + get_importable_name(cand.__class__)
+                graph.add_node(cand_name, cand_type, get_init_parameters_or_fail(cand))
```

Comment:
this subgraph is a little strange because it has no connection, which would be not friendly to graph visualization

Reply:
I don't think base graph should be visualized.

Reply:
ok